var/da/da_setup_structures/da_setup_be_regional.inc

```diff
@@ -121,9 +121,13 @@ subroutine da_setup_be_regional(xb, be, grid)
    real, allocatable   :: regcoeff_chi_u_rh(:,:,:)   ! chi_u/rh   regression coefficient
    real, allocatable   :: regcoeff_t_u_rh(:,:,:)     ! t_u/rh     regression coefficient
    real, allocatable   :: regcoeff_ps_u_rh(:,:)      ! ps_u/rh    regression coefficient
-   real                :: qrain_th_low, qrain_th_high
+   !real                :: qrain_th_low, qrain_th_high
```

Comment:
This seems to be a substantial change to reading binary files. Maybe converting the data to an existing standard would be better.

Reply:
Yes. Now this one is simplified. gen_be_v3 generated files need to be converted to be.dat (GENBE_2.0 format) so that it can be used.